client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -194,6 +177,20 @@ class GoogleAnalyticsForm extends Component {
 					/>
 				) : (
 					<Card className="analytics-settings site-settings__analytics-settings">
+						{ siteIsJetpack &&
+							! showUpgradeNudge && (
```

Comment:
Do we need this `! showUpgradeNudge` check since we're already in the `! showUpgradeNude` condition of the ternary?

See https://github.com/Automattic/wp-calypso/pull/23036/files#diff-d1f7a612f3d01140b70e4b50aea5f4abR168

Reply:
Good catch 👍  Fixed in e724699197a84c2817433fcd741068baffe48e7a.